source/extensions/filters/http/common/jwks_fetcher.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "envoy/api/v2/core/http_uri.pb.h"
+#include "envoy/common/pure.h"
+#include "envoy/upstream/cluster_manager.h"
+
+#include "jwt_verify_lib/jwks.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Common {
+
+class JwksFetcher {
```

Comment:
Original comment from @qiwzhang on #4225 

"Can you add comment to describe its usage? especially if its instance can be re-used for multiple fetching"

Reply:
Fixed